test/test-pgroonga-check.rb

```diff
@@ -148,4 +148,44 @@ class PGroongaCheckTestCase < Test::Unit::TestCase
       end
     end
   end
+
+  sub_test_case "list-broken-indexes" do
+    test "corrupt" do
+      omit("This test takes over 10 minutes, so skip it.") if ENV["CI"]
+
+      run_sql("CREATE TABLE memos (content text);")
+      run_sql("CREATE INDEX memos_content ON memos USING pgroonga (content);")
+      run_sql("INSERT INTO memos SELECT '#{"dummy" * 10}' FROM generate_series(1, 35000000);")
```

Comment:
```suggestion
      run_sql("INSERT INTO memos SELECT repeat('dummy ', 1000) FROM generate_series(1, 350000);")
```

これでギリギリ`.1`ができるくらいのデータなの？

もっとテスト時間を短くしたいねぇ。

Reply:
もう少しギリギリを攻めてみます。

Reply:
insertする文字数は関係ないようす。
0文字でも35,000,000レコードをinsertすると、`.001` ファイルができる。
逆に文字数を増やしてレコード数を減らすと、`.001` ファイルはできない。

ということで、空文字を33,600,000レコード登録するテストに変えました。（33,500,000レコードだと `.001` ファイルは作られず）
ローカル環境だと6.5分くらいです。変更前は10.5分くらい。